core/common/src/main/java/alluxio/security/authorization/AccessControlList.java

```diff
@@ -71,17 +69,14 @@
   public static final String OWNING_GROUP_FIELD = "owningGroup";
   public static final String STRING_ENTRY_FIELD = "stringEntries";
 
-  /** Key representing owning user in {@link #mUserActions}. */
+  /** Keys representing owning user and group for proto ser/de. */
```

Comment:
this comment should apply to the constants above `OWNER_FIELD` , `OWNING_GROUP_FIELD` , `STRING_ENTRY_FIELD`

Reply:
Moved the comment.